src/Account.sol

```diff
@@ -747,30 +797,39 @@ contract Account is IAccount, Auth, OpsReady {
     }
 
     /*//////////////////////////////////////////////////////////////
-                   GELATO CONDITIONAL ORDER HANDLING
+                       EXECUTE CONDITIONAL ORDER
     //////////////////////////////////////////////////////////////*/
 
     /// @inheritdoc IAccount
     function executeConditionalOrder(uint256 _conditionalOrderId)
         external
         override
+        nonReentrant
         isAccountExecutionEnabled
-        onlyOps
     {
-        // store conditional order in memory
+        // store conditional order object in memory
         ConditionalOrder memory conditionalOrder =
```

Comment:
**Suggestion:** Move this below `_validConditionalOrder` check to save gas in the case of failing orders.